crates/ra_ide/src/syntax_highlighting.rs

```diff
@@ -514,13 +540,13 @@ fn highlight_element(
 
             let expr = prefix_expr.expr()?;
             let ty = sema.type_of_expr(&expr)?;
+            let mut h = HighlightTag::Operator.into();
             if !ty.is_raw_ptr() {
-                return None;
+                h
+            } else {
+                h |= HighlightModifier::Unsafe;
+                h
```

Comment:
Is this change still necessary?

Reply:
I wouldn't say it's unnecessary, but it is unrelated to this change. Should I just remove it? It could always be added as a separate PR.

Reply:
For context, this was just ensuring that safe deref operations are marked as an operation. The only other items with this currently are the question mark operator and deref when the operation is unsafe.